app/controllers/admin/sessions_controller.rb

```diff
@@ -0,0 +1,26 @@
+class Admin::SessionsController < Admin::AdminController
+  layout 'admin'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.